src/App.tsx

```diff
@@ -16,9 +16,9 @@ const App = () => {
                 "--background-color": settings.themeBackgroundColor,
                 "--foreground-color": settings.themeForegroundColor,
             } as React.CSSProperties}
-            >
+        >
             <Particles />
-            <img src={settings.themeBackgroundImagePath} />
+            <img src={settings.themeBackgroundImageBase64 ? `data:image/jpeg;base64,` +  settings.themeBackgroundImageBase64 : ""} />
```

Comment:
can you set it so that the image rendered correctly:
1. If no image is configured, then dont need this img at all
2. Add settings to adjust the background: alignment (left bottom / right top / center / etc), size (cover / contains / fit / etc)
3. Feel free to change into css background if its better, I just put random <img> here just for future reminder.